cmd/chaos-builder/impl.go

```diff
@@ -97,6 +97,7 @@ func (in *{{.Type}}) GetChaos() *ChaosInstance {
 		Kind:      Kind{{.Type}},
 		StartTime: in.CreationTimestamp.Time,
 		Action:    "",
+		Status:    string(in.Status.ChaosStatus.Experiment.Phase),
```

Comment:
`in.Status.Experiment.Phase`. we can omit `ChaosStatus`

Reply:
Got it! Already fixed.